bundle/manifests/external-dns-operator.clusterserviceversion.yaml

```diff
@@ -303,9 +303,15 @@ metadata:
     certified: "false"
     containerImage: quay.io/openshift/origin-external-dns-operator:latest
     createdAt: 2021/09/28
+    features.operators.openshift.io/disconnected: "true"
+    features.operators.openshift.io/fips-compliant: "false"
```

Comment:
External DNS can work on FIPS enabled cluster, I think we could set `fips-compliant` as true.

Reply:
Can it? You verified this?

Reply:
Done.

Reply:
Reverted back to `false` as the operator and operand use the static linkage (`CGO_ENABLED=0`) which is [not FIPS compliant](https://docs.google.com/presentation/d/1o3IowxHX6BsnxGkIInaQ0lBgnn_K5Ex8jxwCYCeNsqs/edit#slide=id.g276351f5efb_2_1).